### PR TITLE
java Beginner Chapter 2

### PR DESCRIPTION
**Pull request by Pokosteler**
java Beginner Chapter 2(24,25,26,27 tasks)

### CODE REVIEW

File: src/lesson7/javaBeginnerChapter2/task24/Time.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    private int seconds;
+    private int minutes;
+    private int hours;
+
+    public Time(int seconds) {
+        this.seconds = seconds;
+    }
+
+    public Time(int seconds, int minutes, int hours) {
+        this.seconds = seconds;
```
Comment (RomanKashenok) [inline src/lesson7/javaBeginnerChapter2/task24/Time.java]: Implement logic for conversion one type to another. 
Now, for example, I can create Time(32425, 3254254, 1321314) and it'll be stored exactly this way. But you should to store seconds with value less than 60, and minutes too

File: src/lesson7/javaBeginnerChapter2/task25/ATM.java [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        this.hundredDollars = hundredDollars;
+    }
+
+    public void addMoney(int twentyDollars, int fiftyDollars, int hundredDollars) {
+        this.twentyDollars += twentyDollars;
+        this.fiftyDollars += fiftyDollars;
+        this.hundredDollars += hundredDollars;
+    }
+
+    public Boolean withdrawMoney(int twentyDollars, int fiftyDollars, int hundredDollars) {
```
Comment (RomanKashenok) [inline src/lesson7/javaBeginnerChapter2/task25/ATM.java]: Method should receive one argument. And you need to write logic for appropriate withdraw

File: src/lesson7/javaBeginnerChapter2/task26/LargeAppliance.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package lesson7.javaBeginnerChapter2.task26;
+
+abstract class LargeAppliance extends HomeAppliance {
```
Comment (RomanKashenok) [inline src/lesson7/javaBeginnerChapter2/task26/LargeAppliance.java:3]: You should extend functionality of superclass. But you just create another class similar to superclass:
state and behavior of LargeAppliance  is similar to HomeAppliance

File: src/lesson7/javaBeginnerChapter2/task26/SmallElectricalAppliances.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package lesson7.javaBeginnerChapter2.task26;
+
+abstract class SmallElectricalAppliances extends HomeAppliance {
+    public SmallElectricalAppliances(String name) {
```
Comment (RomanKashenok) [inline src/lesson7/javaBeginnerChapter2/task26/SmallElectricalAppliances.java:6]: The same comment as in previous class

File: src/lesson7/javaBeginnerChapter2/task27/CashCard.java [hunk 5]
```diff
@@ -0,0 +1,7 @@
+package lesson7.javaBeginnerChapter2.task27;
+
+abstract class CashCard extends BankCard {
+    public CashCard(String name) {
```
Comment (RomanKashenok) [inline src/lesson7/javaBeginnerChapter2/task27/CashCard.java:6]: The same as previous comment - you should extend superclass, but not duplicate it

File: src/lesson7/javaBeginnerChapter2/task27/CreditCard.java [hunk 6]
```diff
@@ -0,0 +1,7 @@
+package lesson7.javaBeginnerChapter2.task27;
+
+abstract class CreditCard extends BankCard {
+    public CreditCard(String name) {
```
Comment (RomanKashenok) [inline src/lesson7/javaBeginnerChapter2/task27/CreditCard.java:7]: The same as previous comment - you should extend superclass, but not duplicate it

**Review by RomanKashenok — CHANGES_REQUESTED** [review]

File: src/lesson7/javaBeginnerChapter2/task24/Time.java (hunk 1 above)
Comment (Pokosteler) [reply src/lesson7/javaBeginnerChapter2/task24/Time.java]: fixed

File: src/lesson7/javaBeginnerChapter2/task25/ATM.java (hunk 2 above)
Comment (Pokosteler) [reply src/lesson7/javaBeginnerChapter2/task25/ATM.java]: fixed

File: src/lesson7/javaBeginnerChapter2/task26/LargeAppliance.java (hunk 3 above)
Comment (Pokosteler) [reply src/lesson7/javaBeginnerChapter2/task26/LargeAppliance.java:3]: fixed

File: src/lesson7/javaBeginnerChapter2/task26/SmallElectricalAppliances.java (hunk 4 above)
Comment (Pokosteler) [reply src/lesson7/javaBeginnerChapter2/task26/SmallElectricalAppliances.java:6]: fixed

File: src/lesson7/javaBeginnerChapter2/task27/CashCard.java (hunk 5 above)
Comment (Pokosteler) [reply src/lesson7/javaBeginnerChapter2/task27/CashCard.java:6]: fixed

File: src/lesson7/javaBeginnerChapter2/task27/CreditCard.java (hunk 6 above)
Comment (Pokosteler) [reply src/lesson7/javaBeginnerChapter2/task27/CreditCard.java:7]: fixed

**Pull request merged**